content/blog/microcks-realm-fix-for-apicurio.md

```diff
@@ -0,0 +1,142 @@
+## Microcks Keycloak Realm fixes for APICurio Studio integration

```

Comment:
Please add a metadata section at the top like the one on https://raw.githubusercontent.com/microcks/microcks.io/master/content/blog/microcks-1.1.0-release.md.

This help providing metadatas and information authoring.

Reply:
Sure @lbroudoux,  will add the metadata section.

Reply:
Added metadata section. Also added a bio pic in the static content folder.